### PR TITLE
[v4.1] RavenDB-11543 List terms inside 'from orders where OrderedAt in (|,|,|)'

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Studio/typescript/common/queryCompleter.ts [hunk 1]
```diff
@@ -285,15 +288,10 @@ class queryCompleter {
...
-                    const firstToken = token.value.trim();
-                    if (firstToken !== "" && firstToken !== "," && firstToken !== "." && firstToken !== "[].") {
-                        lastToken = token;
-                        continue;
-                    }
                 }
             }
-
+            
+            console.debug(token.type + ": " + token.value);
```
Comment (ml054) [inline src/Raven.Studio/typescript/common/queryCompleter.ts]: remove it?

File: src/Raven.Studio/typescript/common/queryCompleter.ts [hunk 2]
```diff
@@ -482,7 +487,9 @@ class queryCompleter {
 
         this.session = session;
         this.callback = callback;
+        console.group("RQL Last Keyword");
```
Comment (ml054) [inline src/Raven.Studio/typescript/common/queryCompleter.ts]: remove?

File: src/Raven.Studio/typescript/common/queryCompleter.ts [hunk 3]
```diff
@@ -482,7 +487,9 @@ class queryCompleter {
...
         const lastKeyword = this.lastKeyword = this.getLastKeyword(pos);
+        console.groupEnd();
```
Comment (ml054) [inline src/Raven.Studio/typescript/common/queryCompleter.ts]: remove ?

**Review by ml054 — CHANGES_REQUESTED** [review]
please remove console.log/console.debug

File: src/Raven.Studio/typescript/common/queryCompleter.ts (hunk 1 above)
Comment (fitzchak) [reply src/Raven.Studio/typescript/common/queryCompleter.ts]: Done

File: src/Raven.Studio/typescript/common/queryCompleter.ts (hunk 2 above)
Comment (fitzchak) [reply src/Raven.Studio/typescript/common/queryCompleter.ts]: ok

**Pull request merged**